packages/live-share/src/internals/TestTeamsClientApi.ts

```diff
@@ -48,6 +48,8 @@ export interface FluidTenantInfo {
  */
 export interface TeamsClientApi {
     interactive: TeamsClientApiInteractive;
+    // Teams app namespace
+    app?: any;
```

Comment:
add todo to make sure it gets removed if we're not using it after we release to GA?